atk/sys/Cargo.toml

```diff
@@ -17,7 +17,7 @@ shell-words = "1.0.0"
 tempfile = "3"
 
 [features]
-dox = []
+dox = ["glib/dox", "gobject/dox"]
```

Comment:
Would you mind making sure the dox features are properly set on the safe wrapper crates as well?